### PR TITLE
INTMDB-165: Update connection strings for cluster

### PR DESCRIPTION
**Pull request by gssbzn**
## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Comment by themantissa** [comment]
@coderGo93  and @leofigy  adding you both as this is the client side for INTMDB-158 for Terraform.

File: mongodbatlas/clusters.go [hunk 1]
```diff
@@ -30,8 +30,9 @@ var _ ClustersService = &ClustersServiceOp{}
 
 // AutoScaling configures your cluster to automatically scale its storage
 type AutoScaling struct {
-	DiskGBEnabled *bool    `json:"diskGBEnabled,omitempty"`
-	Compute       *Compute `json:"compute,omitempty"`
+	AutoIndexingEnabled *bool    `json:"autoIndexingEnabled,omitempty"` // Autopilot mode is only available if you are enrolled in the Auto Pilot Early Access program.
```
Comment (themantissa) [inline mongodbatlas/clusters.go:33]: Good add, we'll need to support this in Terraform post beta.

**Review by themantissa — APPROVED** [review]
Thank you for getting this in cluster and for the two other updates.  Tagged DoU as we'll need this work for Terraform as well so would like one of them to ensure they are good with it too. Thanks!

File: mongodbatlas/clusters.go (hunk 1 above)
Comment (gssbzn) [reply mongodbatlas/clusters.go:33]: I did a quick pass to see if there was anything else new we were missing and this was the only other thing

Comment (themantissa) [reply mongodbatlas/clusters.go:33]: Thank you for doing that!

**Review by leofigy — APPROVED** [review]
thanks !

**Pull request merged**